Ably 1.2.16 Fails to install in non-git repo

**tldr:** Version 1.2.16 Breaks `npm install` or `npm ci` if not in a git repo.

**Reproduce**
```shell
mkdir ~/ably-error
cd ~/ably-error
npm init -y
npm install ably@1.2.15
npm install ably@1.2.16
```

**Error Messsage**
```shell
npm ERR! code 128
npm ERR! path ~/ably-error/node_modules/ably
npm ERR! command failed
npm ERR! command sh -c git submodule init && git submodule update
npm ERR! fatal: not a git repository (or any of the parent directories): .git

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2022-01-14T16_24_06_483Z-debug.log
```

**Cause**
Seems to be caused by the `preinstall` script expecting a git repository (https://github.com/ably/ably-js/blob/c97898cf11e641f35316f56964d1df656ec6080e/package.json#L105)

The 1.2.15 -> 1.2.16 diff shows that script was added. (https://github.com/ably/ably-js/compare/1.2.15...1.2.16#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R105)

┆Issue is synchronized with this [Jira Uncategorised](https://ably.atlassian.net/browse/SDK-1485) by [Unito](https://www.unito.io)

--- a/package.json
+++ b/package.json
@@ -101,7 +101,6 @@
     "check-closure-compiler": "grunt check-closure-compiler",
     "prepare": "npm run build",
     "sourcemap": "source-map-explorer browser/static/ably.min.js",
-    "sourcemap:noencryption": "source-map-explorer browser/static/ably.noencryption.min.js",
-    "preinstall": "git submodule init && git submodule update"
+    "sourcemap:noencryption": "source-map-explorer browser/static/ably.noencryption.min.js"
   }
 }

